modules/frontend/service/frontend.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=frontend service
```

Comment:
Name it something more specific.
Frontend service is way too generic to give a hint.
Niffler frontend service makes better sense.